airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -94,6 +95,9 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
     :param full_url_mode: if True, the secrets must be stored as one conn URI in just one field per secret.
         If False (set it as false in backend_kwargs), you can store the secret using different
         fields (password, user...).
+    :param secret_values_are_urlencoded: If True, and full_url_mode is False, then the values are assumed to
```

Comment:
For such boolean value, the verb is/are is always at the beginning

```suggestion
    :param are_secret_values_urlencoded: If True, and full_url_mode is False, then the values are assumed to
```